### PR TITLE
feat: slider

### PR DESCRIPTION
**Pull request by ddivyansh18**
- Added slider
image

### CODE REVIEW

File: Components/Slider/slider.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  width: 10rem;
+}
+
+.slider {
+  appearance: none;
+  width: 100%;
+  height: 0.4rem;
+  border-radius: 0%;
+  background: lightgray;
+  outline: none;
```
Comment (thegeorgenikhil) [inline Components/Slider/slider.css]: Better to use hex code instead of colour names.

File: Components/Navigation/navigation.html [hunk 2]
```diff
@@ -88,10 +88,10 @@ <h1>App</h1>
...
         </nav>
 
-        <br />
-        <br />
-        <br />
         <p>
+          <br />
+          <br />
+          <br />
           Lorem Ipsum is simply dummy text of the printing and typesetting
```
Comment (thegeorgenikhil) [inline Components/Navigation/navigation.html]: Use padding insted of `br` tag as it is not semantic as well as bad for screen readers. [Link](https://stackoverflow.com/questions/1726073/is-it-sometimes-bad-to-use-br#:~:text=The%20main%20reason%20for%20not,space%20the%20blocks%20out%20properly.)

**Review by thegeorgenikhil — APPROVED** [review]
Issues
- hex codes instead of color names
- use of br tags

Everything else looks good to merge! Do the required changes. Approving this.

File: Components/Slider/slider.css (hunk 1 above)
Comment (ddivyansh18) [reply Components/Slider/slider.css]: Thanks for the suggestions ! Just wanted to know for information, why hex codes are better than colour names ?

File: Components/Navigation/navigation.html (hunk 2 above)
Comment (ddivyansh18) [reply Components/Navigation/navigation.html]: Agreed, will fix

**Review by Hnh013 — APPROVED** [review]
looks good to merge !

File: Components/Slider/slider.css (hunk 1 above)
Comment (thegeorgenikhil) [reply Components/Slider/slider.css]: Not sure but I think hex codes are universally supported in all browsers and these color names came up recently so some browser may have issues interpreting these name.

File: Components/Navigation/navigation.html (hunk 2 above)
Comment (ddivyansh18) [reply Components/Navigation/navigation.html]: @thegeorgenikhil  Removed all br tags from the code

File: Components/Slider/slider.css (hunk 1 above)
Comment (ddivyansh18) [reply Components/Slider/slider.css]: Slider background now uses a color variable

**Pull request merged**